include/boost/http_proto/fields_base.hpp

```diff
@@ -265,6 +298,8 @@ class BOOST_SYMBOL_VISIBLE
         <br>
         All iterators that are equal to `before`
         or come after are invalidated.
+        <br/>
+        Leading and trailing whitespace are removed.
```

Comment:
This is ambiguous. Where is the whitespace removed? If there is any already in the container? Nothing is "removed" it is merely not inserted. And if there is an error, this text could mislead the reader into thinking something changed.